src/lib.rs

```diff
@@ -127,3 +127,7 @@ mod win32_surface;
 
 #[cfg(windows)]
 pub use win32_surface::Win32Surface;
+
+mod pdf_surface;
+
+pub use pdf_surface::PDFSurface;
```

Comment:
Please put this export with the others (the `Win32Surface` is a bit of an ugly exception...).